frontend-react/src/pages/error/ErrorPage.tsx

```diff
@@ -1,50 +1,52 @@
-// @ts-nocheck // TODO: fix types in this file
 import { Alert } from "@trussworks/react-uswds";
 import React from "react";
 import Helmet from "react-helmet";
 
 import { NotFound } from "./NotFound";
 import { UnsupportedBrowser } from "./UnsupportedBrowser";
 
-interface ErrorPageProps {
-    code?: string;
-    error?: string;
-    errorInfo?: React.ErrorInfo;
-    type?: "page" | "message";
-}
-
 /* INFO
    For consistency, when passing the code prop, please use these values
    e.g. <ErrorPage code={CODES.NOT_FOUND_404} />
  */
 export enum CODES {
     UNSUPPORTED_BROWSER = "unsupported-browser",
     NOT_FOUND_404 = "not-found",
+    UNKNOWN = "unknown-error",
+}
+
+interface ErrorPageProps {
+    code?: CODES.UNSUPPORTED_BROWSER | CODES.NOT_FOUND_404 | CODES.UNKNOWN;
+    error?: string;
+    errorInfo?: React.ErrorInfo;
+    type?: "page" | "message";
 }
 
-function ErrorPageWrapper({ children }: JSX.Element) {
+const ErrorPageWrapper = (props: React.PropsWithChildren<ErrorPageProps>) => {
```

Comment:
`React.PropsWithChildren` is a nice way to do do this. The `props` then has `props.children` automatically but also can contain OTHER props, too.